README.md

```diff
@@ -1533,6 +1533,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+- [gatling](https://github.com/p-w/gatling-webserver) - a high performance web server. Gatling is a small and fast HTTP webserver and reverse proxy that makes deploying microservices or websites easy. Gatling is particularly good in situations with very high load. ([Source Code](https://www.fefe.de/gatling/)) `GPL-2.0` `C`
```

Comment:
I'd rather add it to https://github.com/awesome-foss/awesome-sysadmin#web, on hold until https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2482 is resolved.